### ISSUE TITLE
[BUG] Unable to use normal browser Download functionality

### ISSUE THREAD

**Issue by AndrewMore1** [Adobe-CEP/CEP-Resources#167]
It appears that we are unable to download files in the usual manner from within the browser.

Currently we trigger a download by calling "window.open(<url to file>, "_self", "enabled");

This works correctly when used in a stand alone browser, but appears to do nothing in the special browser page for the Adobe extension.

Attaching the debugger does seem to indicate some sort of transfer was initiated.  In the debugger console window I get the following message:

Resource interpreted as Document but transferred with MIME type image/png: "http://10.134.211.7:7001/awc/fms/fmsdownload/sailboat.png?ticket=f87923a6d3…%2B%2B%2B%2B%2B%255cengineering_51bb6049%255csailboa_ima_lbi08w87dtsdm.png".

**Comment by codearoni** [comment]
Sounds like the Content-type request header is a document (application/pdf) but the response header is image/png. This mismatch results in that error message.

**Comment by AndrewMore1** [comment]
This is not correct, since that same message (it's not an error, because it's not in red) appears in the stand alone Chromium 41 browser.  Further that message is not displayed at all in later versions of Chrome.
